<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>BotW Save Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabLayout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabEditSave">
     <attribute name="title">
      <string>game_data</string>
     </attribute>
     <widget class="QSpinBox" name="spinboxMaxHeart">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelMaxHeart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Heart</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCurrentHeart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Heart</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinboxCurrentHeart">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCredits">
     <attribute name="title">
      <string>caption</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxMasterMode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Master Mode</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="credits">
     <attribute name="title">
      <string>Credits</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>BotW Save Editor by Mystixor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>GUI by Mika Westphal (IGambleNull)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Big thanks to zephenryus, assisting to reverse engineer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>791</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
